subsys/net/l2/openthread/openthread.c

```diff
@@ -299,42 +299,15 @@ static enum net_verdict openthread_recv(struct net_if *iface,
 
```

Comment:
In commit subject, it is enough to say `net: openthread: ....`, the lib is redundant information here.